plugins/catalog-backend/src/service/CatalogBuilder.ts

```diff
@@ -420,7 +421,9 @@ export class CatalogBuilder {
   /**
    * Wires up and returns all of the component parts of the catalog
    */
-  async build(): Promise<{
+  async build(
+    conflictHandler?: (options: ConflictHandlerOptions) => Promise<void>,
```

Comment:
An alternative approach could be to add this to the CatalogEnv. Not sure which is preferrable

Reply:
The builder already has a large number of setters for different types of adaptations. None of them are on the build method. Any particular reason you put it there?

Reply:
Ah good point yes thats much better